src/test/CMakeLists.txt

```diff
@@ -113,6 +113,8 @@ set(
     tasks/operator_task_test.cpp
     testing_assert.cpp
     testing_assert.hpp
+    testing_utils.cpp
+    testing_utils.hpp
```

Comment:
(random location)
Previously, test files were overwritten in the next run. Where do you make sure that these test files get cleaned up even if the test crashes?

Reply:
Since we are creating random names, I do not see an easy and neat way to clean up if tests crash. On the plus side: we are creating files in `/tmp/`